pkg/archive/jar.go

```diff
@@ -10,6 +10,12 @@ import (
 	"strings"
 )
 
+var AllJarStabilizers []Stabilizer = []Stabilizer{
+	StableJARBuildMetadata,
+	StableJAROrderOfAttributeValues,
+	StableGitProperties,
```

Comment:
ah this should probably be StableJARGitProperties to keep with the naming scheme here. can be done in a follow-on change tho.

Reply:
Fixed [`9b372b2` (#363)](https://github.com/google/oss-rebuild/pull/363/commits/9b372b27ca3c3131f91304265c982af12f710f2b)